docs/index.md

```diff
@@ -0,0 +1,32 @@
+# Jenkins-k8s Operator
+
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing [Jenkins](https://www.jenkins.io/) Agent on machines and configurable to use a Jenkins charm deployed in another Juju model.
+
+This charm simplifies initial deployment and "day N" operations of Jenkins Agent on VMs and bare metal.
+
+As such, the charm makes it easy for those looking to take control of their own agents whilst keeping operations simple, and gives them the freedom to deploy on the platform of their choice.
+
+For DevOps or SRE teams this charm will make operating Jenkins Agent simple and straightforward through Juju's clean interface. It will allow easy deployment into multiple environments for testing changes, and supports scaling out for enterprise deployments.
+
+## Project and community
+
+The Jenkins-agent Operator is a member of the Ubuntu family. It's an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
+
+- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/)
+- [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
+- [Contribute](Contribute)
+
+Thinking about using the Jenkins-k8s Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
+
+# Contents
+
+1. [Tutorial](tutorial)
+   1. [Getting Started](tutorial/getting-started.md)
+1. [How to](how-to)
```

Comment:
Yes, I brought it over from another docs set of another charm. It seems to allow for proper numbering : https://github.com/canonical/saml-integrator-operator/blob/d15b6e2b5bdac7c457ae5759f0cee748674e5c78/docs/index.md

Reply:
Markdown renders usually figure out proper numbering even if you don't bother with it in source. This makes it easier to add/remove items since you don't have to update all the other preceding/ following lines

Reply:
I see, thanks for the explanation. I think in that case there's no change necessary here since numbering is done automatically.